dev/ci/ci-basic-overlay.sh

```diff
@@ -79,8 +79,8 @@
 ########################################################################
 # VST
 ########################################################################
-: ${VST_CI_BRANCH:=less_init_plugins}
-: ${VST_CI_GITURL:=https://github.com/letouzey/VST.git}
+: ${VST_CI_BRANCH:=master}
+: ${VST_CI_GITURL:=https://github.com/Zimmi48/VST.git}
```

Comment:
On the longer term, this should be fixed by asking developments we track to have a branch tracking master and distinct ones for stable releases.